index.html

```diff
@@ -3,17 +3,28 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="Author" content="Hussein" />
+    <meta name="author" content="Hussein" />
```

Comment:
You can use you fullname here

Reply:
### This comment has been taken care of.